src/providers/wms/qgswmsprovider.cpp

```diff
@@ -1809,6 +1809,23 @@ QString QgsWmsProvider::layerMetadata( QgsWmsLayerProperty &layer )
   QString::number( layer.fixedHeight ) %
   QStringLiteral( "</td></tr>" );
 
+  // Metadata URLs
+  if ( !layer.metadataUrl.isEmpty() )
+  {
+    metadata += QStringLiteral( "<tr><th>" ) %
+                tr( "MetadataURLs" ) %
+                QStringLiteral( "</th>"
+                                "<td><table class=\"tabular-view\">"
+                                "<tr><th>Format</th><th>URL</th></tr>" );
```

Comment:
Should format and url be translated too?

Reply:
Right